tasks/70_binary-decode/binary-decode.js

```diff
@@ -0,0 +1,11 @@
+export function binaryDecode(binaryCode) {
+  if (binaryCode === undefined) {
+    return '';
+  }
+  return binaryCode
+      .split(' ')
+      .map((char) => {
+        return String.fromCharCode(parseInt(char, 2));
+      })
```

Comment:
This could be one-liner

```suggestion
      .map((char) => String.fromCharCode(parseInt(char, 2)))
```